CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/svg/SVGObject.java

```diff
@@ -0,0 +1,286 @@
+     */
+    public final String getType() {
+        return this.type;
+    }
+    
+    /**
+     * Adds an attribute to the current SVG element. 
+     * Only one attribute value can be provided per key.
+     * Attempting to add a value to an existing key 
+     * will override the last value provided.
+     * @param attributeKey such as "height", "width", "fill"
+     * @param attributeValue such as "100", "stroke-width:3", "rgb(0,0,255)".
+     */
+    public final void setAttribute(final String attributeKey, final String attributeValue) {
+        this.attributes.put(attributeKey, attributeValue);
+    }
+    
+    /**
+     * Adds an attribute to the current SVG element. 
+     * Only one attribute value can be provided per key.
+     * Attempting to add a value to an existing key 
+     * will override the last value provided.
+     * @param attributeKey 
+     * @param attributeValue 
+     */
+    public final void setAttribute(final String attributeKey, final float attributeValue) {
+        setAttribute(attributeKey, String.format("%s", attributeValue));
+    }
+    
+    /**
+     * Adds an attribute to the current SVG element. 
+     * Only one attribute value can be provided per key.
+     * Attempting to add a value to an existing key 
+     * will override the last value provided.
+     * @param attributeKey 
+     * @param attributeValue 
+     */
+    public final void setAttribute(final String attributeKey, final int attributeValue) {
+        setAttribute(attributeKey, String.format("%s", attributeValue));
+    }
+    
+    /**
+     * Adds an attribute to the current SVG element. 
+     * Only one attribute value can be provided per key.
+     * Attempting to add a value to an existing key 
+     * will override the last value provided.
+     * @param attributeKey 
+     * @param attributeValue 
+     */
+    public final void setAttribute(final String attributeKey, final double attributeValue) {
+        setAttribute(attributeKey, String.format("%s", attributeValue));
+    }
+        
+    /**
+     * Adds a child SVGObject to the current SVG element.
+     * To avoid duplication, this method should only be called by the setParent() method.
+     * Child SVGObjects are stored in HashMap based on their id attribute.
+     * @param child 
+     */
+    private void setChild(final SVGObject child) {
+        final String childID = child.getAttributeValue(SVGAttributeConstant.ID.getKey());
+        this.children.put(childID, child);
+    }
+    
+    /**
+     * Returns an SVGObject with a specified id attribute value.
+     * @param idValue 
+     * @return ArrayList
+     */
+    public SVGObject getChild(final String idValue) {
+        return this.children.get(idValue);
+    }
+    
+    /**
+     * Returns an ArrayList of SVGObjects containing all child SVGObject elements.
+     * @return ArrayList
+     */
+    public List<SVGObject> getAllChildren() {
+        final ArrayList<SVGObject> allChildren = new ArrayList<>();
```

Comment:
can you make the declaration a plain old list? i.e. `List<SVGObject>`? Instantiation is fine